examples/job/expansions/README.md

```diff
@@ -0,0 +1,255 @@
+In a real use case, the processing would be some substantial computation, such as rendering a frame
+of a movie, or processing a range of rows in a database.  The "$ITEM" parameter would specify for
+example, the frame number or the row range.
+
+This Job has two labels. The first label, `app=jobexample`, distinguishes this group of jobs from
+other groups of jobs (these are not shown, but there might be other ones).  This label
+makes it convenient to operate on all the jobs in the group at once.  The second label, with
+key `item`, distinguishes individual jobs in the group.  Each Job object needs to have
+a unique label that no other job has.  This is it.
+Neither of these label keys are special to kubernetes -- you can pick your own label scheme.
+
+Next, expand the template into multiple files, one for each item to be processed.
+
+```console
+# Expand files into a temporary directory
+$ mkdir ./jobs
+$ for i in apple banana cherry
+do
+  cat job.yaml.txt | sed "s/\$ITEM/$i/" > ./jobs/job-$i.yaml
+done
+$ ls jobs/
+job-apple.yaml
+job-banana.yaml
+job-cherry.yaml
+```
+
+Here, we used `sed` to replace the string `$ITEM` with the the loop variable.
+You could use any type of template language (jinja2, erb) or write a program
+to generate the Job objects.
+
+Next, create all the jobs with one kubectl command:
+
+```console
+$ kubectl create -f ./jobs
+job "process-item-apple" created
+job "process-item-banana" created
+job "process-item-cherry" created
+```
+
+Now, check on the jobs:
+
+```console
+$ kubectl get jobs -l app=jobexample -L item
+JOB                   CONTAINER(S)   IMAGE(S)   SELECTOR                               SUCCESSFUL ITEM
+process-item-apple    c              busybox    app in (jobexample),item in (apple)    1          apple
+process-item-banana   c              busybox    app in (jobexample),item in (banana)   1          banana
+process-item-cherry   c              busybox    app in (jobexample),item in (cherry)   1          cherry
+```
+
+Here we use the `-l` option to select all jobs that are part of this
+group of jobs.  (There might be other unrelated jobs in the system that we
+do not care to see.)
+
+The `-L` option adds an extra column with just the `item` label value.
+
+We can check on the pods as well using the same label selector:
+
+```console
+$ kubectl get pods -l app=jobexample -L item
+NAME                        READY     STATUS      RESTARTS   AGE       ITEM
+process-item-apple-kixwv    0/1       Completed   0          4m        apple
+process-item-banana-wrsf7   0/1       Completed   0          4m        banana
+process-item-cherry-dnfu9   0/1       Completed   0          4m        cherry
+```
+
+There is not a single command to check on the output of all jobs at once,
+but looping over all the pods is pretty easy:
+
+```console
+$ for p in $(kubectl get pods -l app=jobexample -o name | cut -f 2 -d "/")
```

Comment:
I'm still disappointed that the version without the cut didn't work for you. I've just checked once again - works perfectly ok.